pkg/image/api/types.go

```diff
@@ -50,6 +50,12 @@ type Image struct {
 	DockerImageManifest string
 	// DockerImageLayers represents the layers in the image. May not be set if the image does not define that data.
 	DockerImageLayers []ImageLayer
+	// DockerImageSignatures provides the signatures on this JWS as opaque blobs.
+	DockerImageSignatures [][]byte
+	// DockerImageManifestMediaType specifies the mediaType of manifest.
+	DockerImageManifestMediaType string
+	// DockerConfigImage is a JSON blob that the runtime uses to set up the container.
+	DockerConfigImage string
```

Comment:
@smarterclayton This is a place to put the config for manifest of schema v2. Usually it is placed on the filesystem as a separate blob. [Manifest v2](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#image-manifest-field-descriptions) has a reference to it, but not the data itself.